# Off Normal Weak Strong
string value
---
int32 errorCode
string errorDescription